addons/account/wizard/account_move_send_wizard.py

```diff
@@ -254,7 +254,8 @@ def _get_sending_settings(self):
     def _update_preferred_settings(self):
         """If the partner's settings are not set, we use them as partner's default."""
         self.ensure_one()
-        if not self.move_id.partner_id.with_company(self.company_id).invoice_sending_method and len(self.sending_methods) == 1:
+        if self.sending_methods and len(self.sending_methods) == 1 \
+            and not self.move_id.partner_id.with_company(self.company_id).invoice_sending_method :
```

Comment:
```suggestion
        if (
            self.sending_methods
            and len(self.sending_methods) == 1
            and not self.move_id.partner_id.with_company(self.company_id).invoice_sending_method
        ):
```

Reply:
............ is that 3 conditions I see ? :see_no_evil: